lib/orbital/models/response.rb

```diff
@@ -79,6 +79,18 @@ def self.orbital_response_params(response)
           :params_tx_ref_num => extract(response, 'tx_ref_num'),
         }
       end
+
+      def first_reference_id
+        params_tx_ref_num
+      end
+
+      def second_reference_id
+        params_order_id
```

Comment:
Shouldn't this be `first_reference_id` instead?

Usually, `first_reference_id` is the gateway reference while `second_reference_id` the downstream reference (processor, bank, etc.). `second_reference_id` would be used for matching records with the bank's records.